lib/formtools/templates/index.js

```diff
@@ -1,12 +1,6 @@
 var fs = require('fs'),
     handlebars = require('handlebars');
 
-var documents = fs.readFileSync(__dirname + '/documents.html', 'utf-8'),
-    overview = {};
-
-overview.summary = fs.readFileSync(__dirname + '/overview/summary.html', 'utf-8');
+var documents = fs.readFileSync(__dirname + '/documents.html', 'utf-8');
```

Comment:
I know the previous code had it, but can we move to `path.resolve` for traversing directories, i.e. (`path.resolve(__dirname, 'documents.html')`).